README.md

```diff
@@ -1,12 +1,51 @@
-# software-quality-handbook
-Software Quality Assurance Assignment
-Project plan – 
-Tasks to complete project and their order
-1.	Add team members as collaborators on the repository – One person
-2.	Members fork repository – Everyone 
-3.	Clone repository to local machine – Everyone
-4.	Divide up sections – Everyone
-5.	Agree on how to review pull requests – Everyone
-6.	Research each topic for the handbook – specific to sections below
-7.	Start adding to repo based on what section we are working on using Gitflow process - Everyone
-8.	readme change
+### Code Reviews
+#### Into
+***
+
+Agile teams are self-organizing, with skill sets that span across the team.
+This is accomplished, in part, with code review.
+Code review helps developers learn the code base, as well as help them learn
+new technologies and techniques that grow their skill sets
+
+#### What is code review
+***
+Code review is a circular process on a specific branch before it reaches the merge phase into dev
+or master.
+The place where the code review is being done is the pull request.
```

Comment:
Maybe here it could read something like "Code reviews are done when a developer makes a pull request."